src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java

```diff
@@ -1174,15 +1174,17 @@ public boolean onWireCutterRightClick(ForgeDirection side, ForgeDirection wrench
     @Override
     public boolean onSolderingToolRightClick(ForgeDirection side, ForgeDirection wrenchingSide, EntityPlayer aPlayer,
             float aX, float aY, float aZ) {
+        super.onSolderingToolRightClick(side, wrenchingSide, aPlayer, aX, aY, aZ);
```

Comment:
```suggestion
if (super.onSolderingToolRightClick(side, wrenchingSide, aPlayer, aX, aY, aZ)) {
    return
}
``` 

This means if superclass did anything, we will bail out early instead of letting one user action translate into two change in the controller

Reply:
Wouldn't it make more sense to only call super if we did nothing here. Because otherwise there could in theory be the situation that we can never do our thing because super always soles something. That would be against the idea of why we do overrides.